sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -159,48 +194,77 @@
       }
     </style>
 
-    <!--Toolbar with buttons such as +Notebook, +Folder... etc-->
-    <!--TODO: [yebrahim] Make these buttons do stuff-->
     <div id="toolbar" hidden$="{{small}}">
-      <paper-button class="toolbar-button" on-click="_createNewNotebook">
-        <iron-icon icon="add-box"></iron-icon>
-        <span>Notebook</span>
-      </paper-button>
-      <paper-button class="toolbar-button" on-click="_createNewFile">
-        <iron-icon icon="add-box"></iron-icon>
-        <span>File</span>
-      </paper-button>
-      <paper-button class="toolbar-button" on-click="_createNewDirectory">
-        <iron-icon icon="add-box"></iron-icon>
-        <span>Folder</span>
-      </paper-button>
-      <paper-button class="toolbar-button">
-        <iron-icon icon="file-upload"></iron-icon>
-        <span>Upload</span>
-      </paper-button>
-      <paper-button class="toolbar-button" on-click="_copySelectedItem"
-                    disabled$={{!selectedFile}}>
-        <iron-icon icon="content-copy"></iron-icon>
-        <span>Copy</span>
-      </paper-button>
-      <paper-button class="toolbar-button" on-click="_moveSelectedItem"
-                    disabled$={{!selectedFile}}>
-        <iron-icon icon="redo"></iron-icon>
-        <span>Move</span>
-      </paper-button>
-      <paper-button class="toolbar-button" on-click="_renameSelectedItem"
-                    disabled$={{!selectedFile}}>
-        <iron-icon icon="editor:border-color"></iron-icon>
-        <span>Rename</span>
-      </paper-button>
-      <!--TODO: Delete operation supports deleting multiple files, but we'll
-      only enable the button when one file is selected for now, until we get
-      the group moving/copying functionality too-->
-      <paper-button class="toolbar-button" on-click="_deleteSelectedItems"
-                    disabled$={{!selectedFile}}>
-        <iron-icon icon="delete"></iron-icon>
-        <span>Delete</span>
-      </paper-button>
+
+      <!--Default Add toolbar, contains Add buttons for notebook, file, and folder-->
+      <span id="addToolbar">
```

Comment:
For consistency with "FileOps", maybe call this "AddOps" instead of just "Add"?
Or maybe rename "FileOps" to something like "UpdateOps", since these Add ops are also file ops.
Or don't use "Ops" in either.